### ISSUE TITLE
Doesnt work

### ISSUE THREAD

**Issue by Anydex** [lucenasoft/creator-instagram-accounts#1]
The code is messy as fuck, this garbage shit needs a lot of work.

**Comment by lucenasoft** [comment]
of course kskskskks, the code is in pt-br and it was my first project, when there is more time in my day to day, I will redo the code, I have this code saved for almost 1 year... dkkdkdkkk so.

**Comment by lucenasoft** [comment]
of course kskskskks, the code is in pt-br and it was my first project, when there is more time in my day to day, I will redo the code, I have this code saved for almost 1 year... dkkdkdkkk so.

**Comment by Anydex** [comment]
wtf is wrong with you lol

**Comment by lucenasoft** [comment]
Yes, soon I'll update, supporting proxy and new features and maybe even an interface I'll add.

**Comment by lucenasoft** [comment]
Yes, soon I'll update, supporting proxy and new features and maybe even an interface I'll add.

**Comment by Anydex** [comment]
No need for an interface, just make it work efficiently and easily,

**Comment by Anydex** [comment]
Thank you

**Comment by Anydex** [comment]
oi, when will u update it!
